### DISCUSSION TITLE
Change the switch to a button

### DISCUSSION THREAD

**Discussion by G-us** [AlexGustafsson/homebridge-wol#196] [Questions] [answered]
I have a special device for restarting my computer but I would rather it be a a button. Is this something you could work on or I could change on my own?

**Comment by AlexGustafsson** [comment, marked as answer]
I'll add this as a suggestion in #123. Basically, you'd like for your device to show up as a button in Home and when pressed, you'd like to use this plugin and send some command to your device to restart the computer?

**Reply by G-us** [reply]
Yes exactly, thanks!
